NORDIC Embedded Software/nrf52 Patient Monitoring Device/HRS_file.c

```diff
@@ -98,7 +99,7 @@ uint32_t MAX30105_Get_Sample (void)
     {
         MAX30105_set_pointer(MAX30105_FIFO_DATA);
         nrf_drv_twi_rx(&MAX30105_twi, MAX30105_ADD, data, 6);
-        //data from 0 - 2 indexes is red and we don't care
+        //data from 0 - 2 indexes is irrelevant, but must be read
         IR[3] = 0 ;
         IR[2] = data[3];
         IR[1] = data[4];
```

Comment:
Consider to remove nrf_log from 109 line. It is commented, probably You do not want to use it. *be pragmatic*